### DISCUSSION TITLE
havok plugin

### DISCUSSION THREAD

**Discussion by unicomp21** [RaananW/babylonjs-webpack-es6#37] [General]
@RaananW is the havok plugin open source?

**Comment by RaananW** [comment]
The engine itself is not open source, but free to use under the MIT license

**Comment by unicomp21** [comment]
Thanks!  Is the wasm api for havok fully exposed via typescript?  Or is there another set of docs I should be reading?  I'm curious if there is other stuff in the wasm module, like navmesh etc.

**Reply by RaananW** [reply]
The wasm API has typescript typings to help you develop with it, but it is not fully documented. You can see how the babylon plugin is using it if you are trying to use it yourself, otherwise you can simply use babylon's physics architecture that uses havok under the hood
